packages/db/test/cli/gen-migration.test.mjs

```diff
@@ -0,0 +1,45 @@
+import { once } from 'events'
+import { execa } from 'execa'
+import fs from 'fs'
+import split from 'split2'
+import { test } from 'tap'
+import { cliPath, getFixturesConfigFileLocation } from './helper.mjs'
+
+test('generates next file correctly', async ({ equal, match }) => {
+  const configFilePath = getFixturesConfigFileLocation('auto-apply.json')
+  const buffer = await fs.promises.readFile(configFilePath)
+  const config = JSON.parse(buffer.toString())
+
+  const migrationsBefore = await fs.promises.readdir(getFixturesConfigFileLocation(config.migrations.dir))
+  await execa('node', [cliPath, 'generate', 'migration', '-c', configFilePath])
+  const migrationsAfter = await fs.promises.readdir(getFixturesConfigFileLocation(config.migrations.dir))
+
+  const newMigrations = migrationsAfter.filter(migrationFile => !migrationsBefore.includes(migrationFile))
+  equal(newMigrations.length, 2)
+  match(newMigrations[0], '.do.sql')
+  match(newMigrations[1], '.undo.sql')
+
+  await Promise.all(newMigrations.map(migration => fs.promises.unlink(getFixturesConfigFileLocation(`${config.migrations.dir}/${migration}`))))
+})
+
+test('throws if migrations directory does not exist', async ({ match }) => {
+  const child = execa('node', [cliPath, 'generate migration', '-c', getFixturesConfigFileLocation('invalid-migrations-directory.json')])
+  const output = child.stderr.pipe(split())
+  const [data] = await once(output, 'data')
+  match(data, /^MigrateError: Migrations directory (.*) does not exist.$/)
+})
+
+// test('throws if migration name is wrong', async ({ match, teardown}) => {
```

Comment:
This test hangs up and I need some help to understand why.

If you run the same command locally:
```

node ./packages/db/db.mjs generate migration -c ./packages/db/test/fixtures/gen-migration-bad-name.json
Previous migration has corrupted name
```

The output is correct:
```
Previous migration has corrupted name
```

Reply:
I would await the object returned by the call to execa and just verify that the stderr is correct.

Reply:
Thanks for the tip, that worked out.